frontend/src/pages/snippet/FileToolbar.jsx

```diff
@@ -96,7 +96,8 @@ function SnippetName({ snippet }) {
             ref={inputRef}
             as={AutowidthInput}
             autoComplete="off"
-            className="transition-padding"
+            className="transition-padding text-center"
+            style={{ minWidth: '70px' }}
```

Comment:
@zebpaa we should limit ourselves to using inline styles. Try adding a custom class. There are probably ready-made classes in bootstrap. Why did you decide to use centering?

Reply:
@fey Without centering it doesn't look very nice, in my opinion (attached screenshot). I could reduce the minimum width, but the problem discussing in issue, as I understand it, indicates that the input is too short.
![without_text-center](https://github.com/user-attachments/assets/a621328b-a00e-4fd2-ae49-58f80a00f685)

I found the work of one of the developers and decided to do it in a similar way, including min-w-70.